data-structures/hash-tables.md

```diff
@@ -1,11 +1,11 @@
 # Hash Tables
 
 ### Projected Time
-2 hours
+2 hours and 30 minutes
 
 ### Prerequisites
-JavaScript prototypal inheritance
-Runtime complexity
+- [JavaScript prototypal inheritance](/javascript)
+- [Runtime complexity](/runtime-complexity/runtime-complexity.md)
 
```

Comment:
(/javascript) -> (../javascript)
(/runtime-complexity/runtime-complexity.md) -> (../runtime-complexity/runtime-complexity.md)

Reply:
Either way works

Reply:
Can you specify which JS lesson plz

Reply:
> Can you specify which JS lesson plz

It was already there. I have not touched it. But I will find it if you suggest.

Reply:
Can you help us find which JS lesson shows inheritance?